docs/software-engineering-policies/SourceControl/SourceControlPolicy.md

```diff
@@ -1,3 +1,55 @@
 # Source Control
 
-(Under construction)
\ No newline at end of file
+## Repositories
```

Comment:
Worth bullet pointing these lists for readability?

Reply:
is that better?